### PR TITLE
fix syntax error

### PR DESCRIPTION
**Pull request by stevenspiel**

### CODE REVIEW

**Comment by stevenspiel** [comment]
3.0.5

File: lib/src/util/crypto_util.dart [hunk 1]
```diff
@@ -82,7 +82,7 @@ UuidValue nameUuidFromBytes(List<int> name) {
   final bytes = md5.convert(name).bytes;
   bytes[6] = (bytes[6] & 0x0f) | 0x30;
   bytes[8] = (bytes[8] & 0x3f) | 0x80;
-  return UuidValue.fromList(bytes);
+  return UuidValue.fromList(Uint8List.fromList(bytes));
```
Comment (boyan01) [inline lib/src/util/crypto_util.dart:85]: the `bytes` from `md5.convert(name).bytes` should be `List<int>`
and  `UuidValue.fromList(List<int> byteList, {int? offset})` also accept `List<int>` as a parameter.

There should be no syntax errors here.

Comment (stevenspiel) [reply lib/src/util/crypto_util.dart]: whoops. I put the fix on the wrong line. Updated it.

**Comment by boyan01** [comment]
SGTM

dart 3.1  has a break change for this  https://github.com/dart-lang/sdk/issues/52801

**Pull request merged**